src/chainparams.cpp

```diff
@@ -378,8 +382,8 @@ class CRegTestParams : public CChainParams {
         consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nStartTime = 0;
         consensus.vDeployments[Consensus::DEPLOYMENT_TESTDUMMY].nTimeout = Consensus::BIP9Deployment::NO_TIMEOUT;
 
-        consensus.nMinimumChainWork = uint256S("0x00");
```

Comment:
fa82143e41c067e7d16d612c775701be94974d54

Not sure what is the motivation of being explicit here, I'd just remove these cases.

Reply:
https://github.com/bitcoin/bitcoin/pull/20035#issuecomment-700700349